mingw-w64-git/PKGBUILD

```diff
@@ -187,7 +187,6 @@ package_git () {
 
   # Git Bash, Git CMD
   install -d -m755 $pkgdir$SHAREDIR
-  install -m755 ../git-for-windows.ico $pkgdir$SHAREDIR
```

Comment:
With a proper Sign-off, and after you verified that it works, I'll gladly accept it ;-)